.github/workflows/generate_issues.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       with:
         access-token: '${{ secrets.GITHUB_TOKEN }}'
         # This is to exclude workflows that will be searched in the nightly report.
-        exclude-workflow-files: 'performance-integration-tests.yml, sessions-integration-tests.yml'
+        exclude-workflow-files: 'performance-integration-tests.yml, sessions-integration-tests.yml, codeql'
```

Comment:
It looks like the casing on the workflow is CodeQL. https://github.com/firebase/firebase-ios-sdk/actions/workflows/github-code-scanning/codeql  Does that matter?

Reply:
I'm going to resolve and merge to try out.